src/main/java/frc/robot/subsystems/drive/Drive.java

```diff
@@ -30,22 +30,20 @@
 import edu.wpi.first.wpilibj.DriverStation.Alliance;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
+import frc.robot.subsystems.gyro.Gyro;
 import frc.robot.util.LocalADStarAK;
 import org.littletonrobotics.junction.AutoLogOutput;
 import org.littletonrobotics.junction.Logger;
 
 public class Drive extends SubsystemBase
 {
-    private final GyroIO                   _gyroIO;
-    private final GyroIOInputsAutoLogged   _gyroInputs = new GyroIOInputsAutoLogged();
+    Gyro                                   _gyro; // TODO: Check implementation
```

Comment:
Implementation is good, we just need some modifiers:

```suggestion
    private final Gyro                     _gyro;
```